sdk/storage/azblob/README.md

```diff
@@ -0,0 +1,362 @@
+handle(err)
+```
+
+For more information about blob service URL's and how to configure custom domain names for Azure Storage check out the [official documentation][azure_portal_account_url]
+
+#### Types of credentials
+
+The clients support different forms of authentication. Storage accounts can use a Shared Key Credential, Connection
+String, or a Shared Access Signature Token for authentication
+in [`azidentity`](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity)
+like `azidentity.NewDefaultAzureCredential()`.
+
+The azblob package supports any of the types that implement the `azcore.TokenCredential` interface, authorization via a
+Connection String, or authorization with a Shared Access Signature Token.
+
+##### 1. Creating the client from a shared key
+
+To use an account [shared key][azure_shared_key] (aka account key or access key), provide the key as a string. This can
+be found in your storage account in the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or by
+running the following Azure CLI command:
+
+```bash
+az storage account keys list -g MyResourceGroup -n MyStorageAccount
+```
+
+Use AAD authentication as the credential parameter to authenticate the client:
+```golang
+cred, err := azidentity.NewDefaultAzureCredential(nil)
+handle(err)
+serviceClient, err := aztables.NewServiceClient("https://<myAccountName>.table.core.windows.net/", cred, nil)
+handle(err)
+```
+
+##### 2. Creating the client from a connection string
+
+You can use connection string, instead of providing the account URL and credential separately, for authentication as well. 
+To do this, pass the connection string to the client's `NewServiceClientFromConnectionString` method. 
+The connection string can be found in your storage account in the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or with the following Azure CLI command:
+
+```bash
+az storage account show-connection-string -g MyResourceGroup -n MyStorageAccount
+```
+
+```golang
+connStr := "DefaultEndpointsProtocol=https;AccountName=<myAccountName>;AccountKey=<myAccountKey>;EndpointSuffix=core.windows.net"
+serviceClient, err := azblob.NewServiceClientFromConnectionString(connStr, nil)
+```
+
+##### Creating the client from a SAS token
+
+To use a [shared access signature (SAS) token][azure_sas_token], provide the token as a string. 
+If your account URL includes the SAS token, replace credential parameter with `azcore.NewAnonymousCredential()`. 
+If your account URL includes the SAS token, replace credential parameter with `azcore.NewAnonymousCredential()`. 
+You can generate a SAS token from the Azure Portal under [Shared access signature](https://docs.microsoft.com/rest/api/storageservices/create-service-sas) or use
+the `ServiceClient.GetSASToken` or `ContainerClient.GetSASToken()` methods.
+
+```go
+credential, err := azblob.NewSharedKeyCredential("accountName", "accountKey")
+if err != nil {
+log.Fatal(err)
```

Comment:
```suggestion
    log.Fatal(err)
```

Reply:
This applies in a few other spots in this sample

Reply:
Though GO wouldn't mind, I think it should look good too. So +1